docs/ModelZoo.md

```diff
@@ -0,0 +1,1024 @@
+# NNCF Compressed Model Zoo
+
+Here we present the results achieved using our sample scripts, example patches to third-party repositories and NNCF configuration files.
+
+- [PyTorch](#pytorch)
+  * [Classification](#pytorch-classification)
+  * [Object Detection](#pytorch-object-detection)
+  * [Semantic Segmentation](#pytorch-semantic-segmentation)
+  * [Natural Language Processing (3rd-party training pipelines)](#pytorch-nlp-huggingface-transformers-powered-models)
+- [TensorFlow](#tensorflow)
+  * [Classification](#tensorflow-classification)
+  * [Object Detection](#tensorflow-object-detection)
+  * [Instance Segmentation](#tensorflow-instance-segmentation)
+- [ONNX](#onnx)
+
+## PyTorch
+
+### PyTorch Classification
+
+<table>
+    <thead>
+		<tr>
+			<th style="text-align: center;" colspan="6"><br>Quantization<br>&nbsp</th>
```

Comment:
I agree that it looks cleaner, but results in a anchor symbol appearing when hovering over the header: 
![image](https://github.com/openvinotoolkit/nncf/assets/23343961/f7626408-1971-43e8-a0ce-302153321cfd)

I'd keep it as is because it looks the same and does not have this disadvantage

Reply:
We decided to remove intermediate header for compression algorithms, and state "Quantization" explicitly